### ISSUE TITLE
/.pomerium/ visual endpoint not working since v0.17.0

### ISSUE THREAD

**Issue by sylr** [pomerium/pomerium#3356]
## What happened?

Since I've upgraded all pomerium components with version >= v0.17.0 the /.pomerium/ HTML endpoint which displays identity information does not work (it redirects me to the the authentication provider for login).

FYI downgrading the authorization component back to v0.16.4 resolves the problem.

## How'd it happen?

1. Ran `x`
2. Clicked `y`
3. Saw error `z`

## What's your environment like?

- Pomerium version (retrieve with `pomerium --version`): v0.17.3
- Server Operating System/Architecture/Cloud: kubernetes

## What's your config.yaml?

Authentication

```config.yaml
---
log_level: info
address: 0.0.0.0:8080
insecure_server: true
grpc_insecure: true
idp_refresh_directory_interval: 5m
idp_refresh_directory_timeout: 5m
forward_auth_url: https://sso-forward-auth.company.io
authenticate_service_url: https://sso-auth.company.io
authorize_service_url: http://pomerium-authorize.sso.svc.cluster.local.:8081
databroker_service_url: http://pomerium-cache.sso.svc.cluster.local.:8081
metrics_address: 0.0.0.0:9090
idp_provider: azure
idp_provider_url: https://login.microsoftonline.com/xxxxx-xxxx-xxx-xxxxx-xxxxxx/v2.0
```

Proxy:

```
---
autocert: false
log_level: info
address: 0.0.0.0:8080
insecure_server: true
grpc_address: 0.0.0.0:8081
grpc_insecure: true
forward_auth_url: https://sso-forward-auth.company.io
authenticate_service_url: https://sso-auth.company.io
authorize_service_url: http://pomerium-authorize.sso.svc.cluster.local.:8081
databroker_service_url: http://pomerium-cache.sso.svc.cluster.local.:8081
metrics_address: 0.0.0.0:9090
idp_provider: azure
idp_provider_url: https://login.microsoftonline.com/xxxxx-xxxx-xxx-xxxxx-xxxxxx/v2.0
jwt_claims_headers: email,groups,user
cookie_domain: .company.io
xff_num_trusted_hops: 1
routes: ...
```

## What did you see in the logs?

```logs
# Paste your logs here.
# Be sure to scrub any sensitive values
```

**Comment by desimone** [comment]
@sylr -- Would you mind recording a loom or creating a `har` file so we can take a look at the flow? 


Can you share your forward auth config as well / downstream proxy details? 

Thanks!

**Comment by sylr** [comment]
Fixed in v0.19.1.
